components/File/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,141 @@
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="File" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <a
+      href="/example.pdf"
+      class="denhaag-file"
+      aria-labelledby="name"
+      aria-describedby="description"
+      download="Example.pdf"
+    >
+      <div class="denhaag-file__left">
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+          <path
+            d="M13 2.5H6C5.46957 2.5 4.96086 2.71071 4.58579 3.08579C4.21071 3.46086 4 3.96957 4 4.5V20.5C4 21.0304 4.21071 21.5391 4.58579 21.9142C4.96086 22.2893 5.46957 22.5 6 22.5H18C18.5304 22.5 19.0391 22.2893 19.4142 21.9142C19.7893 21.5391 20 21.0304 20 20.5V9.5L13 2.5Z"
+            stroke="#4B4B4B"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path d="M13 2.5V9.5H20" stroke="#4B4B4B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+        </svg>
+      </div>
+      <div class="denhaag-file__right">
+        <div class="denhaag-file__label">
+          <span id="name">File name </span>
+          <span id="description">(pdf, 2MB, 1-1-2023)</span>
+        </div>
+        <div class="utrecht-link utrecht-link--icon-left" tabindex="-1">
+          Download
+        </div>
+      </div>
+    </a>
+  </Story>
+  {/* eslint-enable react/no-unknown-property */}
+</Canvas>
+
+### Hover
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Hover">
+    <a
+      href="/example.pdf"
+      class="denhaag-file denhaag-file--hover"
+      aria-labelledby="name"
+      aria-describedby="description"
+      download="Example.pdf"
+    >
+      <div class="denhaag-file__left">
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+          <path
+            d="M13 2.5H6C5.46957 2.5 4.96086 2.71071 4.58579 3.08579C4.21071 3.46086 4 3.96957 4 4.5V20.5C4 21.0304 4.21071 21.5391 4.58579 21.9142C4.96086 22.2893 5.46957 22.5 6 22.5H18C18.5304 22.5 19.0391 22.2893 19.4142 21.9142C19.7893 21.5391 20 21.0304 20 20.5V9.5L13 2.5Z"
+            stroke="#4B4B4B"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path d="M13 2.5V9.5H20" stroke="#4B4B4B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+        </svg>
+      </div>
+      <div class="denhaag-file__right">
+        <div class="denhaag-file__label">
+          <span id="name">File name </span>
+          <span id="description">(pdf, 2MB, 1-1-2023)</span>
+        </div>
+        <div class="utrecht-link utrecht-link--icon-left" tabindex="-1">
```

Comment:
I think we want the icon as svg element inside the download link, like in the Den Haag link: https://nl-design-system.github.io/denhaag/?path=/docs/css-navigation-link--link-with-icon-before#link-with-icon-before-hovered, because I think is somewhat more flexible and in the React component we could use the <DownloadIcon /> component from the @gemeente-denhaag/icons package. But i see Utrecht does this with an icon in the CSS, @Yolijn what's your opinion about this?